jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -240,5 +233,24 @@ private static Path getConfigDir() {
     return XdgDirectories.getConfigHome();
   }
 
+  /**
+   * Attempts an atomic move first, and falls back to non-atomic if the file system does not support
+   * atomic moves.
+   *
+   * @param source the source path
+   * @param destination the destination path
+   * @throws IOException if the move fails
+   */
+  private static void tryAtomicMove(Path source, Path destination) throws IOException {
+    // Attempts an atomic move first, and falls back to non-atomic if the file system does not
+    // support atomic moves.
```

Comment:
nit: unnecessary comments given the Javadoc

Reply:
Whoops, thought I deleted this.